Assets/Scripts/Actors/ActorComponents/PlayerActorResources.cs

```diff
@@ -83,7 +82,7 @@ void CheckScroll()
 				nextIndex = numResources - ((-nextIndex) % numResources);
 			}
 			
-			resourceIndex = Mathf.FloorToInt(Mathf.Clamp(nextIndex, 0, heldResourceTypes.Count - 1));
+			resourceIndex = Mathf.FloorToInt( Mathf.Clamp(nextIndex, 0, heldResourceTypes.Count - 1) );
```

Comment:
It looks like the FloorToInt call is unnecessary because Mathf.Clamp is overloaded to return an int if an int is passed in.